.github/workflows/test-application.yaml

```diff
@@ -0,0 +1,82 @@
+name: Test application
+
+on:
+    pull_request:
+    push:
+        branches:
+            - master
+            - '[0-9]+.[0-9]+'
```

Comment:
as written in https://github.com/handcraftedinthealps/RestRoutingBundle/pull/9/files#r553375508 and as reference https://github.com/sulu/sulu-docs/blob/3962e881c89d5dc6bfe76e4c8bd32985c0dfb6fa/.github/workflows/build-docs.yaml#L7

I would here add also:

```
           - '[0-9]+.x'
```

If we in future maybe change the branching module to we don't need to change the github actions again.

Reply:
fixed